clap_complete_nushell/tests/nushell.rs

```diff
@@ -12,6 +12,18 @@ fn basic() {
     );
 }
 
+#[test]
+fn multi_line() {
```

Comment:
Can you split up the commits
- First commit adds the tests with them passing, showing the buggy behavior
- Second commit fixes the bug and updates the tests so they still pass

This is a way of helping to test the test and makes the intended behavior change clearer to reviewers.